CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ### citus v11.0.2 (June 15, 2022) ###
 
+* Drops support for PostgreSQL 12
+
 * Open sources enterprise features, see the rest of changelog items
 
+* Turns metadata syncing on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
```

Comment:
I think we should remove ` Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to Citus 11+ from earlier versions`

It is just an internal function on 11.0

Reply:
How about updating the next line with some information from here?

My suggestion:
* Introduces `citus_finish_citus_upgrade()` procedure which is necessary to upgrade from earlier versions?

Reply:
looks good